pubspec.yaml

```diff
@@ -1,18 +1,23 @@
 name: sass_builder
-version: 2.0.2
+version: 2.0.3
+
 authors:
 - Luis Vargas <luisvargastijerino@gmail.com>
 - Kevin Moore <kevmoo@github.com>
 - Nicholas Shahan <nshahan@github.com>
+
 homepage: https://github.com/dart-league/sass_builder
 description: Transpile sass files using the "build" package.
+
 environment:
   sdk: '>=2.0.0-dev.51 <2.0.0'
+
 dependencies:
   build: ^0.12.5
-  build_config: ^0.2.6
+  build_config: ^0.3.0
```

Comment:
[optional] This would not have been breaking for this usage, so it would be safe to use `">=0.2.6 <0.4.0"`

That said, pretty much everyone is going to want the `0.3.x` versions anyway so it isn't likely to make much difference.